src/app/components/inline-form/index.html

```diff
@@ -0,0 +1,155 @@
+<sky-demo-page
+  pageTitle="Inline form"
+  skyuxModule="SkyInlineFormModule"
+  packageName="@skyux/inline-form"
+>
+
+<sky-demo-page-summary>
+	<p>
+		The inline form component renders a form in context in the current view rather than opening it in a separate modal. It is most commonly used in <a stacheRouterLink="/components/grid">grids</a>, <a stacheRouterLink="/components/repeater">repeaters</a>, and <a stacheRouterLink="/components/tile">tiles</a>.
+	</p>
+</sky-demo-page-summary>
+
+<sky-demo-page-properties sectionHeading='Inline form properties'>
+	<sky-demo-page-property
+    propertyName="config"
+  >
+  Specifies configuration options for the buttons to display with the inline form. This property accepts <a stacheRouterLink="." fragment="skyinlineformconfig-properties">a <stache-code>SkyInlineFormConfig</stache-code> object</a>.
+  </sky-demo-page-property>
+	<sky-demo-page-property
+    propertyName="showForm"
+  >
+  Specifies the form to display inline.
```

Comment:
Specifies when the inline form should display (or something like that). In other words, this property toggles the display of the form.
Type: boolean.